jmxtrans-output/jmxtrans-output-core/src/main/java/com/googlecode/jmxtrans/model/output/StatsDWriter.java

```diff
@@ -180,6 +195,10 @@ public void internalWrite(Server server, Query query, ImmutableList<Result> resu
 
 	private synchronized boolean doSend(String stat) {
 		try {
+			if (isDebugEnabled()) {
```

Comment:
Please use placeholders and remove the `if (isDebugEnabled())`. See [slf4j manual](http://www.slf4j.org/manual.html#typical_usage) for details.